ketcher-autotests/tests/File-Management/Open-And-Save-Files/MOL-Files/mol-files-export-import-query-features.spec.ts

```diff
@@ -0,0 +1,1592 @@
+    page,
+  }) => {
+    /*
+        Test case: https://github.com/epam/ketcher/issues/3860 - Case 5
+        Description: User can export and load back molecules with "All Atom Query features together" query feature to Mol file. Feature values remain in place.
+        1. Clear canvas
+        2. Load All Atom Query features together.ket
+        3. Validate canvas
+        4. Export canvas to SDF V2000 file
+        5. Clear canvas
+        6. Load SDF V2000 file
+        7. Validate canvas
+        */
+    await openFileAndAddToCanvasAsNewProject(
+      'KET/Query-Features/All Atom Query features together.ket',
+      page,
+    );
+    await moveMouseAway(page);
+    await takeEditorScreenshot(page);
+
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await clickOnFileFormatDropdown(page);
+    await page.getByTestId('SDF V2000-option').click();
+
+    const sdfFileV2000Expected = page
+      .getByTestId('sdf-preview-area-text')
+      .inputValue();
+    //  Save it to test-data if no file in where
+    await saveToFile(
+      'SDF/Query-Feature/All Atom Query features together-expected.sdf',
+      await sdfFileV2000Expected,
+    );
+    await pressButton(page, 'Cancel');
+
+    await openFileAndAddToCanvasAsNewProject(
+      'SDF/Query-Feature/All Atom Query features together-expected.sdf',
+      page,
+    );
+    await moveMouseAway(page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('36. All Atom Query features together: User can export and load back molecules with query feature to RXN V2000 file. Feature values remain in place.', async ({
+    page,
+  }) => {
+    /*
+        Test case: https://github.com/epam/ketcher/issues/3860 - Case 6
+        Description: User can export and load back molecules with "All Atom Query features together" query feature to RXN V2000 file. Feature values remain in place.
+        1. Clear canvas
+        2. Load All Atom Query features together2.ket
+        3. Validate canvas
+        4. Export canvas to RXN V2000 file
+        5. Clear canvas
+        6. Load RXN V2000 file
+        7. Validate canvas
+        */
+    await openFileAndAddToCanvasAsNewProject(
+      'KET/Query-Features/All Atom Query features together2.ket',
+      page,
+    );
+    await moveMouseAway(page);
+    await takeEditorScreenshot(page);
+
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await clickOnFileFormatDropdown(page);
+    await page.getByTestId('MDL Rxnfile V2000-option').click();
+
+    const rxnFileV2000Expected = page
+      .getByTestId('rxn-preview-area-text')
+      .inputValue();
+    //  Save it to test-data if no file in where
+    await saveToFile(
+      'Rxn-V2000/Query-Feature/All Atom Query features together2-expected.rxn',
+      await rxnFileV2000Expected,
+    );
+    await pressButton(page, 'Cancel');
+
+    await openFileAndAddToCanvasAsNewProject(
+      'Rxn-V2000/Query-Feature/All Atom Query features together2-expected.rxn',
+      page,
```

Comment:
Looks like all 36 tests are very similar. Can we run it in loop to reduce the code duplication?